docs/product-manuals/clients/go-client/index.md

```diff
@@ -60,13 +60,13 @@ func main() {
 
 Let's go over this code snippet line by line:
 
-- First we create the credentials provider for the OAuth protocol. This is needed to authenticate your client.
-- Next we create the client passing in the address of the cluster we want to connect to, as well as, the credentials provider from the step above.
-- Finally, we send a test request to verify that the connection was established.
+1. First, we create the credentials provider for the OAuth protocol. This is needed to authenticate your client.
+2. Next, we create the client by passing in the address of the cluster we want to connect to and the credentials provider from the step above.
+3. Finally, we send a test request to verify the connection was established.
```

Comment:
Thoughts on removing "first", "next", and "finally" since we have the numbered steps? Feels a bit redundant to me.

Reply:
Agreed. Will adjust here.